examples/thirdweb/components/NFTDropCard/NFTDropCard.tsx

```diff
@@ -59,6 +59,7 @@ type Props = {
   className?: string
   contractAddress?: string
   chainId?: string
+  clientId?: string
```

Comment:
This `clientId` prop doesn't seem to be used by the `NFTDropCard` component

Reply:
Let me take a look @migueloller

Reply:
Hey @migueloller removed the unused `clientId` from the `NFTDropCard` component.